ignite/templates/typed/map/map.go

```diff
@@ -159,7 +210,7 @@ func protoRPCModify(opts *typed.Options) genny.RunFn {
 					"google.api.http",
 					fmt.Sprintf(
 						"/%s/%s/%s/%s",
-						appModulePath, opts.ModuleName, typenameSnake, indexPath,
+						appModulePath, opts.ModuleName, typenameSnake, protoIndex,
```

Comment:
maybe we can use `filepath.Join` here

Reply:
no, as it isn't really a file system path, but more like an url